wao/util.py

```diff
@@ -1,6 +1,11 @@
 EXECUTION_PATH = '/root/workspace2/execution'  # do not move this to config
 from wao.config import Config
 import sys
+import time
+import watchdog
+import threading
+
+from wao._429_watcher import _429_Watcher
```

Comment:
remove redundant imports 

@Bemhreth I have mentioned many times before to review your own pr first to avoid these kinds of things. Please review your own pr, before I start reviewing. This will save us a lot of time.